src/uct/cuda/cuda_copy/cuda_copy_md.c

```diff
@@ -286,52 +439,119 @@ static size_t uct_cuda_copy_md_get_total_device_mem(CUdevice cuda_device)
     return 1; /* return 1 byte to avoid division by zero */
 }
 
+static ucs_status_t
+uct_cuda_copy_detect_vmm(const void *address, CUmemorytype *cuda_mem_type,
```

Comment:
maybe return 1 or 0 if it is VMM or not?

Reply:
That would not catch the case where memory is detected to be vmm but there was an error getting allocation properties [here](https://github.com/openucx/ucx/pull/9867/files#diff-f990528a14758f5a40bc106bd5fb5888513d667e4fc0b80256fe09dee83015c0R468)